ReactiveCocoa/UIKit/UITextView.swift

```diff
@@ -7,6 +7,11 @@ extension Reactive where Base: UITextView {
 	public var text: BindingTarget<String> {
```

Comment:
`UITextView` text property is optional, so yes, `String?` would be closer to the truth, the same with label and textfield. Do you want me to make the change?